site/_guides/proxy-proto.md

```diff
@@ -20,21 +20,9 @@ Your services should see the addresses in the `X-Forwarded-For` or `X-Envoy-Exte
 
 ## Enable PROXY protocol on your service in AWS
 
-To instruct EC2 to place the ELB into `tcp`+`PROXY` mode, add the following annotations to the `contour` Service:
+To instruct EC2 to place the ELB into `tcp`+`PROXY` mode, follow the steps mentioned in this [AWS documentation][2]
```

Comment:
I haven't looked at this in a while, but doesn't this still work with the old ("classic") ELBs? Just not with NLBs? If that's the case, then this documentation should reflect that.

Just want to make sure that we're not leaving people who are using other configurations behind here.

Reply:
If you don't have time to check if it works with ELBs, that's fine, could you update so that it's clear this is talking about NLBs?

Reply:
@youngnick fixed it. Thanks!